src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,78 @@
+.clock {
+  @extend %center-object;
```

Comment:
8. [BEM & STYLES] - Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.